src/Debug.hs

```diff
@@ -50,15 +50,18 @@ module Debug(
     debugView, debugSave, debugPrint,
     -- * Clear a trace
     debugClear,
+    -- * Exported for tests only
+    removeLet,
+    removeExtraDigits
```

Comment:
The `Debug` module name is very important to keep small, so I suggest breaking these into `Debug.Util` or similar and importing them into `Debug` and the test.

Reply:
Ok, I'll do that tonight